packages/graphql/src/graphql/directives/fulltext.ts

```diff
@@ -43,6 +45,13 @@ export const fulltextDirective = new GraphQLDirective({
                             fields: {
                                 type: new GraphQLNonNull(new GraphQLList(GraphQLString)),
                             },
+                            queryName: {
+                                type: GraphQLString,
+                            }
```

Comment:
Oops, small syntax issue here!

```suggestion
                            },
```